services/api-dotnet/README.md

```diff
@@ -1,50 +1,53 @@
+
+## Projects
+
+|Name|Description|
+|:---|:----------|
+|[API (NoSQL version)](src/app/Api.NoSql)|.NET backend starter, configured to work with MongoDB|
+|[API (SQL version)](src/app/Api.Sql)|.NET backend starter, configured to work with PostgreSQL|
+|[API models](src/app/Api.Views)|The project to share models, mappings and validators across APIs|
+|[Common](src/app/Common)|DAL (for MongoDB and PostgresSQL) and some configuration, which can be shared across all applications|
+|[Services](src/app/Common.Services)|Domain/infrastructure services, shared across all applications|
+|[Scheduler](src/app/Scheduler)|Hangfire scheduler|
+|[Websocket server](src/app/SignalR)|SignalR websocket server (works with MongoDB only)|
+|[Unit tests for NoSQL API](src/app/Tests.NoSql)|Unit tests for NoSQL API|
+|[Unit tests for SQL API](src/app/Tests.Sql)|Unit tests for SQL API|
+
+## Features
+
+- configured console logging ([Serilog](https://serilog.net/))
+- request data validation ([FluentValidation](https://fluentvalidation.net/))
+- object mapping ([Automapper](https://automapper.org/))
+- email sender implementation for local development (can be used with [smtp4dev](https://github.com/rnwood/smtp4dev), for example)
+- Redis cache (see [CacheController](src/app/Api.NoSql/Controllers/CacheController.cs) for example of usage)
+- health checks
+- [Hangfire](https://www.hangfire.io/) scheduler. Can be configure to use either MongoDB or PostgreSQL storage. Dashboard is available on http://localhost:3001/hangfire
+- Feature flags support ([FeatureManagement](https://github.com/microsoft/FeatureManagement-Dotnet)). See [FeatureController](src/app/Api.NoSql/Controllers/FeatureController.cs) for example of usage)
+- Docker support
+- production ready account API resource (singup, signin, forgot password, reset password functionality)
+- access token based authentication
+- database migrations (SQL version only)
+- SignalR websocket server (NoSQL version only)
 
 ## How to set it up
 
-In order to make Paralct.Ship work with this API server on Windows some changes in files are needed to be made.
-
-1. docker-compose.yml:
-	- Add named volume "mongodata" (This will allow to save data even if container is removed)
-	- For mongo service replace "- /var/run/docker.sock:/var/run/docker.sock" with "- mongodata:/data/db" (The path /var/run/docker.sock does not exist on Windows)
-	- Remove api service entirely (.Net API server starts separately)
-	- Add mongodata volume declaration like "db-data" volume here:
-	https://docs.docker.com/compose/compose-file/#endpoint_mode
+Typically a new project will need either SQL or NoSQL DB, so it'll be necessary to make a few changes.
 
-2. web/src/server/config/environment/development.js:
-	- Change jwt secret setting to "jwt_secret128bits" (in .Net it needs to be more than 127 bist long)
+In case of MongoDB:
+- remove Api.Sql project
```

Comment:
Numbered list fits here better to my mind. As this is kind of instruction, not raw list.

Reply:
Also thinking if it makes sense to integrate the last point about `docker-compose.yml` and `appsettings.*.json` files into these instructions too 🤔

Reply:
I'll remove most of these steps by configuring create-ship-app to build a new solution with relevant folders / docker-compose.yml / configs / etc.